pkg/scalers/prometheus_scaler.go

```diff
@@ -266,6 +278,14 @@ func (s *prometheusScaler) ExecutePromQuery(ctx context.Context) (float64, error
 		url = fmt.Sprintf("%s&namespace=%s", url, s.metadata.namespace)
 	}
 
+	for queryParameterKey, queryParameterValue := range s.metadata.queryParameters {
```

Comment:
Is this really necessary? what if we have a parameter like this: `"key=value&??"` ? Wouldn't it be better to not trim input params, but escape them? ie. use `url_pkg.QueryEscape()` as it is done a few lines above

Reply:
thanks for the sugestion, it wasn't necessary, now we are just escaping the query parameters that are sent